aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -0,0 +1,126 @@
+
+import { FormContainer } from '.';
+import { emailMask, emailValidation, phoneMask, states } from './formHelpers';
+
+export const ShippingExample = () => {
+  const [formValues, setFormValues] = React.useState({});
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Shipping
+          </Heading>
+          <Text>for your HPE products</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            validate="blur"
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <Box>
+              <Heading level={4} margin={{ bottom: 'small' }}>
+                Shipping Information
+              </Heading>
+              <Text>Shipping Address</Text>
+              <FormField htmlFor="firstName" name="firstName">
+                <TextInput
+                  id="firstName"
+                  name="firstName"
+                  placeholder="First Name"
+                />
+              </FormField>
+              <FormField htmlFor="lastName" name="lastName">
+                <TextInput
+                  id="lastName"
+                  name="lastName"
+                  placeholder="Last Name"
+                />
+              </FormField>
+              <FormField htmlFor="address1" name="address1">
+                <TextInput
+                  id="address1"
+                  name="address1"
+                  placeholder="Street Address"
+                />
+              </FormField>
+              <FormField htmlFor="address2" name="address2">
+                <TextInput
+                  id="address2"
+                  name="address2"
+                  placeholder="Apt., Suite, Building (Optional)"
+                />
+              </FormField>
+              <FormField htmlFor="city" name="city">
+                <TextInput id="city" name="city" placeholder="City" />
+              </FormField>
+              <FormField htmlFor="state" name="state">
+                <Select
+                  id="state"
+                  name="state"
+                  dropHeight="small"
+                  options={states}
+                  placeholder="Select State"
+                />
+              </FormField>
+              <FormField htmlFor="zipcode" name="zipcode">
+                <TextInput id="zipcode" name="zipcode" placeholder="Zipcode" />
+              </FormField>
+              <CheckBox name="isBusiness" label="This is a business" />
```

Comment:
I'm noticing in the designs, a new checkbox design is being introduced that doesn't match the others. This is a comment of reconciling the theme (which design are we supporting, would we support both?)
Current implementation (matches what is in the theme now): 
<img width="226" alt="Screen Shot 2020-03-06 at 12 12 12 PM" src="https://user-images.githubusercontent.com/12522275/76119181-bc332d80-5fa3-11ea-9436-dceef183be51.png">

This design calls for:
<img width="456" alt="Screen Shot 2020-03-06 at 12 12 36 PM" src="https://user-images.githubusercontent.com/12522275/76119198-c7865900-5fa3-11ea-9208-c8fbf4636632.png">

Reply:
3rd bullet here: https://github.com/hpe-design/design-system/issues/458

Also, the checkbox styling has changed over the last two days, this one got missed.